workflow/scripts/goatools-go-enrichment-analysis.py

```
@@ -93,7 +93,7 @@
     )
     go_terms["study_items"] = go_terms["study_items"].str.join(",")
     go_terms["study_items"] = go_terms.study_items.str.replace(
-        "\w+(?=,|$)", lambda m: ensembl_id_to_symbol.get(m.group(0))
+        "\w+(?=,|$)", lambda m: ensembl_id_to_symbol.get(m.group(0)), regex=True
```

Ruff (0.16.1)

[warning] Invalid escape sequence: `\w`

Use a raw string literal

(W605)